### ISSUE TITLE
Error when extending to new region. getting Invalid Index error related to IAM groups

### ISSUE THREAD

**Issue by alihushamme** [oracle-quickstart/oci-cis-landingzone-quickstart#125]
Hello ,
I am getting multiple invalid Index errors when I am trying to extend to new region. Please note first region deployment worked fine.
I enabled below in quickstart-input.tfvars
extend_landing_zone_to_new_region = true
use_enclosing_compartment           = true
existing_enclosing_compartment_ocid = "ocid1XXXXXXXXX"

Error syntax:
```
│ Error: Invalid index
│ 
│   on iam_groups.tf line 261, in locals:
│  261:   storage_admin_group_name       = length(var.existing_storage_admin_group_name) == 0 && length(trimspace(var.rm_existing_storage_admin_group_name)) == 0               ? [module.lz_groups.groups[local.storage_admin_group_key].name]        : (length(regexall("^ocid1.group.oc.*$", var.rm_existing_storage_admin_group_name)) > 0        ? ["'${data.oci_identity_group.existing_storage_admin_group[var.rm_existing_storage_admin_group_name].name}'"]        : [ for i,v in var.existing_storage_admin_group_name : (length(regexall("^ocid1.group.oc.*$", var.existing_storage_admin_group_name[i])) > 0              ? "'${data.oci_identity_group.existing_storage_admin_group[v].name}'"        :  "'${v}'")])
│     ├────────────────
│     │ local.storage_admin_group_key is "XYZ-storage-admin-group"
│     │ module.lz_groups.groups is object with no attributes
│ 
│ The given key does not identify an element in this collection value.
```

**Comment by andrecorreaneto** [comment]
Hello alihushamme,
we are aware of this and are releasing a fix this week. The workaround is deploying using existing groups when extending the LZ to the other region.
Thanks,
Andre.

**Comment by alihushamme** [comment]
Thanks Andre,
Yes that is same workaround I did yesterday and seem worked. looking forward for your fix release.
FYI below is what I added to quickstart file which seem made things working.
```
# ------------------------------------------------------
# # ----- IAM - Groups
# #-------------------------------------------------------
existing_iam_admin_group_name           = [“XYZ-iam-admin-group"] 
existing_cred_admin_group_name          = ["XYZ-cred-admin-group"] 
existing_security_admin_group_name      = ["XYZ-security-admin-group"] 
existing_network_admin_group_name       = ["XYZ-network-admin-group"] 
existing_appdev_admin_group_name        = ["XYZ-appdev-admin-group"] 
existing_database_admin_group_name      = ["XYZ-database-admin-group"] 
existing_auditor_group_name             = ["XYZ-auditor-group"] 
existing_announcement_reader_group_name = ["XYZ-announcement-reader-group"]
existing_cost_admin_group_name          = ["XYZ-cost-admin-group"] 
existing_storage_admin_group_name       = ["XYZ-storage-admin-group"]
```

**Comment by andrecorreaneto** [comment]
This is fixed in v2.7.0.
